core/domain/stats_domain.py

```diff
@@ -584,13 +644,36 @@ def __init__(self, answer, frequency):
         self.frequency = frequency
 
     def to_raw_type(self):
+        """Returns Python dict for specific answer.
```

Comment:
Returns a Python dict representing the specific answer.

Reply:
Done!